omod/src/main/java/org/openmrs/module/aijar/page/controller/MarkPatientDeadPageController.java

```diff
@@ -41,27 +44,28 @@ public void get(@SpringBean PageModel pageModel, @RequestParam(value = "breadcru
         }
     }
 
-    public String post(@RequestParam(value = "causeOfDeath",required = false) String causeOfDeath, @RequestParam(value = "dead",required = false) Boolean dead, @RequestParam(value = "deathDate",required = false) Date deathDate, @RequestParam("patientId") String patientId, UiUtils
```

Comment:
Its the only way how we can redirect to a patient  dashboard after an even from this method.